iceoryx_hoofs/test/stresstests/test_stress_sofi.cpp

```diff
@@ -20,9 +20,9 @@
 #include "iceoryx_hoofs/testing/test.hpp"
 
 #include <cstdint>
```

Comment:
We didn't update the copyright year for all the other clang-tidy fix warning PRs so I think it is fine to skip it here as well. I do not see any change besides the warning removal.

Reply:
There are code changes though. Copyright is about code changes. We actually should not track the years in the first place, but if we decide to, we have to do it right. Otherwise it is pointless.